lang/lua/Portfile

```diff
@@ -15,23 +15,36 @@ long_description    Lua is a powerful, light-weight programming language designe
                     extending applications. Lua is also frequently used as a general-purpose, \
                     standalone language.
 
-homepage            http://www.lua.org/
+homepage            https://www.lua.org/
 master_sites        ${homepage}ftp/
 
-checksums           rmd160  f260c6f60139cb1464d87b5a7d9673e3f305f368 \
-                    sha256  0c2eed3f960446e1a3e4b9a1ca2f3ff893b6ce41942cf54d5dd59ab4b3b058ac \
-                    size    303543
+checksums           rmd160  cb188956351b40f7a1420067e90d309cdcf8ea45 \
+                    sha256  eac0836eb7219e421a96b7ee3692b93f0629e4cdb0c788432e3d10ce9ed47e28 \
+                    size    349308
 
 patchfiles          patch-Makefile.diff \
                     patch-src-Makefile.diff \
                     patch-src-luaconf.h.diff
 
 # TODO: fixme (a different livecheck is needed for different subports)
-livecheck.type  regex
-livecheck.url   ${master_sites}
-livecheck.regex {lua-(\d+(?:\.\d+)*)}
+livecheck.type      regex
```

Comment:
Minor and not extremely important nitpicking. We try to make whitespace changes into a separate commit.